lib/ReactViews/Tools/DeltaTool/delta-tool.scss

```diff
@@ -0,0 +1,112 @@
+@import "~terriajs-variables";
+
+.deltaTool {
+  composes: tool from "../../tool-panel.scss";
+}
+
+.title {
+  border-bottom: 1px solid $border-color;
+  padding: $padding;
+  font-size: $font-size-mid-small;
+  font-weight: bold;
+}
+
+.body {
+  padding: $padding;
+}
+
+.buttons {
+  display: flex;
+}
+
+.cancelBtn {
+  composes: btn-grey from "../../../Sass/common/_buttons.scss";
+  width: 10vw;
```

Comment:
Can you change this to em please, it doesn't need to change width awkwardly on width-resizing